tasktiger/task.py

```diff
@@ -579,7 +579,12 @@ def tasks_from_queue(
     @classmethod
     def queue_from_function(cls, func: Any, tiger: "TaskTiger") -> str:
         """Get queue from function."""
-        return getattr(func, "_task_queue", tiger.config["DEFAULT_QUEUE"])
+        # Can't use the default here because the attribue might exist, but be
+        # None. In that case we would *not* use the default, which is wrong.
+        attr = getattr(func, "_task_queue", None)
```

Comment:
`func` is now always the `TaskCallable`, right? Should we update the typing and change this to `tc._task_queue or `tiger.config["DEFAULT_QUEUE"]`?